frontend/packages/dev-console/src/components/import/image-search/ImageStream.tsx

```diff
@@ -1,192 +1,185 @@
+            }
             fullWidth
-            disabled={values.imageStream.namespace === ''}
+            disabled={!isNamespaceSelected || (isNamespaceSelected && !isStreamsAvailable)}
             required
+            onChange={() => {
+              setFieldValue('imageStream.tag', '');
+              setFieldValue('isi', initialValues.isi);
+            }}
           />
           <div className="odc-imagestream-separator">/</div>
         </div>
         <div className="col-lg-4 col-md-4 col-sm-4 col-xs-12">
           <DropdownField
             name="imageStream.tag"
             label="Tag"
-            items={getImageStreamTags(values.imageStream.image)}
-            title="Tag"
-            disabled={values.imageStream.image === ''}
+            items={imageStreamTagList}
+            title={
+              isNamespaceSelected && isImageStreamSelected && !isTagsAvailable
+                ? 'No Tag'
+                : 'Select Tag'
+            }
+            disabled={!isImageStreamSelected}
             fullWidth
             required
+            onChange={(tag) => {
+              tag !== '' && searchImageTag(tag);
+            }}
           />
           <div className="odc-imagestream-separator">:</div>
         </div>
       </div>
-      {values.registry === registryType.Internal &&
-        imageNamespace.value &&
-        imageNamespace.value !== builderImagesNamespace.Openshift &&
-        values.project.name !== imageNamespace.value && (
-          <div className="row odc-imagestream-alert">
-            <div className="col-lg-12">
-              <Alert
-                variant="warning"
-                title="add cluster policy to use internal imagestream"
-                isInline
-              >
-                Service account <strong>default</strong> will need image pull authority to deploy
-                images from <strong>{imageNamespace.value}</strong>. You can grant authority with
-                the command:
-                <p>
-                  <code>
-                    oc policy add-role-to-user system:image-puller system:serviceaccount:
-                    {values.project.name}:default -n {imageNamespace.value}
-                  </code>
-                </p>
-              </Alert>
-            </div>
+      {isNamespaceSelected && isImageStreamSelected && !isTagsAvailable && hasCreateAccess && (
+        <div className="col-lg-12 odc-imagestream-alert">
+          <Alert variant="warning" title="No Image streams tags found" isInline>
+            No image streams tags are available in {imageStream.image}
+          </Alert>
+        </div>
+      )}
+      {isNamespaceSelected && !isStreamsAvailable && hasCreateAccess && (
+        <div className="col-lg-12 odc-imagestream-alert">
+          <Alert variant="warning" title="No Image streams found" isInline>
+            No image streams are available in {imageStream.namespace}
```

Comment:
```suggestion
            No image streams are available in project {imageStream.namespace}.
```

I am referring: https://docs.google.com/document/d/1OxnQE-49dkww_6x_0u1bMKi5j1VGggka7zL78hMrxgM/edit?ts=5d920365#heading=h.jykc4dkvr8hr